src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/ZonesSelection.tsx

```diff
@@ -16,18 +16,15 @@ import {PreassignedFareProduct} from '@atb/reference-data/types';
 
 import {Edit} from '@atb/assets/svg/mono-icons/actions';
 import {ThemeIcon} from '@atb/components/theme-icon';
+import {Root_PurchaseTariffZonesSearchByMapScreenParams} from '@atb/stacks-hierarchy/navigation-types';
 
 type ZonesSelectionProps = {
   fareProductTypeConfig: FareProductTypeConfig;
   fromTariffZone: TariffZoneWithMetadata;
   toTariffZone: TariffZoneWithMetadata;
   preassignedFareProduct: PreassignedFareProduct;
-  onSelect: (t: {
-    fromTariffZone: TariffZoneWithMetadata;
-    toTariffZone: TariffZoneWithMetadata;
-    fareProductTypeConfig: FareProductTypeConfig;
-    preassignedFareProduct: PreassignedFareProduct;
-  }) => void;
+  selectionMode: 'single' | 'multiple';
+  onSelect: (t: Root_PurchaseTariffZonesSearchByMapScreenParams) => void;
```

Comment:
as suggestion, perhaps change the name `t` to something more clear like `params` it might be confused with the `t` for language.